terraform-dev/bigquery/page.sql

```diff
@@ -28,14 +28,34 @@ organisations AS (
   WHERE link_type = 'organisations'
   GROUP BY expanded_links.from_url
 ),
-hyperlinks AS (
+all_links AS (
+  SELECT
+    from_url as url, 
+    to_url as link_url
+  FROM
+    content.expanded_links
+  UNION ALL
   SELECT
     url,
-    -- Use DISTINCT because some links are in the table multiple times with
-    -- different link_text
-    ARRAY_AGG(DISTINCT link_url) AS hyperlinks
-  FROM content.embedded_links
-  GROUP BY url
+    link_url
+  FROM
+    content.embedded_links
+), 
+unique_links AS (
+  SELECT DISTINCT 
+    url, 
+    link_url
+  FROM 
+    all_links
+), 
+links AS (
+  SELECT
+    url, 
+    ARRAY_AGG(link_url) as link_url
+  FROM 
+    unique_links
+  GROUP BY
+    url
```

Comment:
I think this could be done in a single step.

```suggestion
links AS (
  SELECT
    url, 
    ARRAY_AGG(DISTINCT link_url) as link_url
  FROM 
    all_links
  GROUP BY
    url
```